pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -262949,7 +262949,6 @@ self: {
        platforms = lib.platforms.x86;
        hydraPlatforms = lib.platforms.none;
        mainProgram = "scat";
-       broken = true;
```

Comment:
So that this change does not get overwritten, please remove `scat` from `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml` and run `maintainers/scripts/haskell/regenerate-transitive-broken-packages.sh`.

Reply:
Ah right, I forgot about this.